css-grid-1/Overview.bs

```diff
@@ -3773,10 +3773,44 @@ Grid Container Baselines</h3>
  ██████  ████ ████████ ████ ██    ██  ██████
 -->
 
-<h2 id='layout-algorithm'>
-Grid Sizing</h2>
+<h2 id='layout-algorithm' oldids="algo-overview" dfn export>
+Grid Layout Algorithm</h2>
 
-	This section defines the <dfn export>grid sizing algorithm</dfn>,
+	This section defines the <b>grid layout algorithm</b>,
+	which sizes the [=grid container=],
+	sizes and positions all the [=grid tracks=],
+	and lays out the [=grid items=]
+	which have been [=grid item placement algorithm|placed=] into its [=grid areas=].
+
+	1. First, find the size of the [=grid container=],
+		per [[#intrinsic-sizes]].
+
+		Note: During this phase,
+		cyclic <<percentage>> track sizes are treated as ''grid-template-columns/auto''.
```

Comment:
Maybe include `fit-content(<percentage>)` too? E.g.

```suggestion
		cyclic track sizing functions containing a <<percentage>> are treated as ''grid-template-columns/auto''.
```

Reply:
I think, given it's a note, that might be unnecessarily specific. :)

Reply:
OK, yeah, it's fine